torchchat/export.py

```diff
@@ -238,7 +238,7 @@ def _to_core_aten(
             raise ValueError(
                 f"Expected passed in model to be an instance of fx.GraphModule, got {type(model)}"
             )
-        core_aten_ep = export(model, example_inputs, dynamic_shapes=dynamic_shapes)
+        core_aten_ep = export_for_training(model, example_inputs, dynamic_shapes=dynamic_shapes)
```

Comment:
Not sure what we are doing here, but shouldn't TorchChat be exporting for inference?

Reply:
This was picked up from @tugsbayasgalan's PR migrating away from export(), but export_for_inference does sound  more in line with what we want

@tugsbayasgalan Can you share info on the new APIs?

Reply:
Yep the intended use for inference IR is that user will export to a training IR and call run_decompositions() to lower to inference IR. In this flow, after core_aten_ep, there is to_edge call which lowers to inference. Export team is moving the IR to non-functional training IR so export_for_training will exist as an alias to official export. After we actually migrate official export, we will replace this call with export.